rust/connlib/shared/src/linux/etc_resolv_conf.rs

```diff
@@ -1,11 +1,14 @@
-use connlib_shared::{
-    linux::{ETC_RESOLV_CONF, ETC_RESOLV_CONF_BACKUP},
-    Error, Result,
-};
+use crate::{Error, Result};
 use std::{net::IpAddr, path::Path};
 use tokio::io::AsyncWriteExt;
 
-/// Back up `/etc/resolve.conf` and then modify it in-place
+pub const ETC_RESOLV_CONF: &str = "/etc/resolv.conf";
+pub const ETC_RESOLV_CONF_BACKUP: &str = "/etc/resolv.conf.firezone-backup";
```

Comment:
```suggestion
pub const ETC_RESOLV_CONF_BACKUP: &str = "/etc/resolv.conf.orig";
```

I don't feel strongly, but I've sometimes seen `.orig` used as a convention for backing up an original file

Reply:
My concern if we don't brand it is, what if some other program like `apt` uses `.orig` and it confuses the programs or even the user? I saw one program do `before-$BRAND` so I may last-minute change it to `.before-firezone` assuming nobody has deployed this on any kitten systems yet